x-pack/test/functional/apps/discover/feature_controls/discover_security.ts

```diff
@@ -190,6 +206,44 @@ export default function (ctx: FtrProviderContext) {
         await PageObjects.unifiedFieldList.expectMissingFieldListItemVisualize('bytes');
       });
 
+      it('should allow for copying the snapshot URL', async function () {
```

Comment:
Not really an issue since the coverage is still there, but just curious why this test was moved from `_shared_links.ts` to this file?

Reply:
Hi @davismcphee good question, in order to test the long url I needed a user without the short url permissions.